src/c20_server/job_translator.py

```diff
@@ -63,7 +66,8 @@ def json_to_job(json_job):
     Record = namedtuple('Record', 'job_id job_type')
     input_array_line = [job_id, job_type]
     record = Record(*input_array_line)
-
+    if record.job_type == NONE_JOB:
```

Comment:
After review it, I believe that we won't have any case that is having 'none' object from json to job object. We only have 'none' that is from job-to-json which used to indicate that there is no job to be done. 

I think we simply delete this statement as well as the create_none_job()